src/main.rs

```diff
@@ -163,22 +166,28 @@ static CHIP_VARS: &[(Chip, &[(&str, &str)])] = &[
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 struct Args {
+    /// Name of the project to generate
     name: String,
 
+    /// Chip to target
     #[arg(short, long)]
     chip: Chip,
 
+    /// Run in headless mode (i.e. do not use the TUI)
     #[arg(long)]
     headless: bool,
 
+    // TODO: Can we list the options and/or point users to some documentation?
```

Comment:
This is probably worth an issue, readme has the list of options but not sure if that is the best place...

Reply:
Created #32